MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,13 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.87.0 ➞ v0.88.0
+### snowflake_tag_association resource changes
+#### *(behavior change)* `object_name`
+No longer deprecated.
+#### *(behavior change)* `object_identifier`
+No longer required.
```

Comment:
Could you elaborate a bit? Why we decided to make this change and what should users use after this change, `object_name` or `object_identifier`? Also, If `object_name` is no longer deprecated then users should be able to use it. Right now, I don't think it's used anywhere else than in schema.

Reply:
reverted change. This PR already has a lot of changes, so probably best not to add even more changes. But I think that the object_identifier struct was not a good idea, and should never have been implemented in the first place. We should only allow for fully-qualified-names as we do for many other resources. However, this kind of change will impact other resources as well which should be changed at the same time.